imod/mf6/drn.py

```diff
@@ -163,6 +165,21 @@ def _validate(self, schemata, **kwargs):
 
         return errors
 
+    def cleanup(
+        self, dis: Union[StructuredDiscretization, VerticesDiscretization]
```

Comment:
Instead of the Union statement you van also write:
`dis: StructuredDiscretization | VerticesDiscretization`

But Union is fine aswel